HomeWork/Program.cs

```diff
@@ -1,12 +1,56 @@
 ﻿using System;
 
-namespace HomeWork
+namespace ConsoleApp2
 {
     class Program
     {
+
+        static void ChangeMinMaxValues(int[] Array)
```

Comment:
Please create independent functions for find min max values. It can be one function which find min/max and return array which contains of [min,max].